ui/components/app/detected-token/detected-token-values/index.scss

```
@@ -0,0 +1,6 @@
+.detected-token-values {
+  &__checkbox {
+    position: absolute;
+    right: 12px;
```

I think you can probably just use margin-left: auto here? And make the parent `width: 100%`

```suggestion
 margin-left: auto;
```

Otherwise you will get some funky errors with this component


https://user-images.githubusercontent.com/8112138/161152183-3146eea8-bd07-49d8-a29b-5827764c0244.mov

---

I was not sure how to tackle the issue I was just doing some trial and error :sweat_smile: I tried your fix but when we click `+10more` link, the checkbox moves with it, I am worried it might look awkward when we have a list of that components. Is there any way I can make wrap the token list without moving the checkbox?


https://user-images.githubusercontent.com/43930900/161166027-64723fb7-e4a0-4978-9b0b-c23e37c9d829.mov